pyproject.toml

```diff
@@ -0,0 +1,49 @@
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = [ "setuptools>=45", "setuptools_scm[toml]>=6.2",]
+
+[project]
+classifiers = [ "Programming Language :: Python :: 3",]
+description = "Ophyd Device definitions for LCLS Beamline components"
+dynamic = [ "version", "readme", "dependencies", "optional-dependencies",]
+keywords = []
+name = "pcdsdevices"
+requires-python = ">=3.9"
+
+[options]
+zip_safe = false
+include_package_data = true
+
+[[project.authors]]
+name = "SLAC National Accelerator Laboratory"
+
+[project.license]
+file = "LICENSE.md"
+
+[project.scripts]
+
+[tool.setuptools_scm]
```

Comment:
This line from the output raised a flag and confused me but might be nothing:
https://github.com/pcdshub/pcdsdevices/actions/runs/3907251443/jobs/6676280720#step:11:800
```
Created wheel for pcdsdevices: filename=pcdsdevices-0.0.0-py3-none-any.whl
```

Reply:
This is only a mid step in the conda builds- in the pip builds it never throws out a 0.0.0

Reply:
Another good catch - this may be an actual issue... 

I thought I had resolved these in the cookiecutter, but perhaps not... I'll double-check the build output for the latest update.

Reply:
No, something's missing in this repository, hmm: https://github.com/pcdshub/cookiecutter-pcds-python/actions/runs/3907026295/jobs/6675800569#step:13:210